### PR TITLE
feat: open questions section

### PR DESCRIPTION
**Pull request by fampinheiro**

### CODE REVIEW

File: packages/react-components/src/organisms/QuestionsSection.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  firstName,
+  questions = [],
+}) => {
+  return (
+    <Card>
+      <Headline2 styleAsHeading={3}>
+        {firstName ? `${firstName}'s Open Questions` : 'Open Questions'}
+      </Headline2>
+      <div css={containerStyles}>
+        {questions.reduce((acc, question, idx) => {
```
Comment (jeysal) [inline packages/react-components/src/organisms/QuestionsSection.tsx:24]: So I think the easiest way I've found to do this (IMO the one we should use) is
```
questions.flatMap(question => [/*Divider*/, /*Question*/]).slice(1)
```
which should be vastly shorter than this

File: packages/react-components/src/templates/__tests__/ProfileResearch.test.tsx [hunk 2]
```diff
@@ -32,10 +33,29 @@ it('renders the skills', () => {
...
+  );
+  expect(getByText(/open questions/i)).toBeVisible();
+  expect(getByText('Q: What is the meaning of life?')).toBeVisible();
+});
+
+it('does not render an empty questions list', () => {
+  const { queryByText } = render(
+    <ProfileResearch {...commonProps} skills={[]} />,
+  );
+  expect(queryByText('Open Questions')).not.toBeInTheDocument();
```
Comment (jeysal) [inline packages/react-components/src/templates/__tests__/ProfileResearch.test.tsx:60]:
```suggestion
  expect(queryByText(/open questions/i)).not.toBeInTheDocument();
```

File: packages/react-components/src/templates/__tests__/ProfileResearch.test.tsx [hunk 3]
```diff
@@ -32,10 +33,29 @@ it('renders the skills', () => {
...
 });
+
+it('renders the questions list', () => {
+  const { getByText } = render(
+    <ProfileResearch
+      {...commonProps}
+      questions={['What is the meaning of life?']}
+    />,
+  );
+  expect(getByText(/open questions/i)).toBeVisible();
```
Comment (jeysal) [inline packages/react-components/src/templates/__tests__/ProfileResearch.test.tsx:52]:
```suggestion
  expect(getByText(/open questions/i).tagName).toBe('H2');
```

File: packages/react-components/src/templates/__tests__/ProfileResearch.test.tsx [hunk 4]
```diff
@@ -32,10 +33,29 @@ it('renders the skills', () => {
...
+  expect(getByText('Q: What is the meaning of life?')).toBeVisible();
```
Comment (jeysal) [inline packages/react-components/src/templates/__tests__/ProfileResearch.test.tsx:53]:
```suggestion
  expect(getByText('What is the meaning of life?', { exact: false })).toBeVisible();
```

File: packages/react-components/src/templates/__tests__/ProfileResearch.test.tsx [hunk 5]
```diff
@@ -32,10 +33,29 @@ it('renders the skills', () => {
...
+});
+
+it('does not render an empty questions list', () => {
+  const { queryByText } = render(
+    <ProfileResearch {...commonProps} skills={[]} />,
```
Comment (jeysal) [inline packages/react-components/src/templates/__tests__/ProfileResearch.test.tsx:58]:
```suggestion
    <ProfileResearch {...commonProps} questions={[]} />,
```

**Review by jeysal — CHANGES_REQUESTED** [review]
FE part mostly LGTM, couple of small comments

**Review by johnytiago — APPROVED** [review]
Reviewed only the BE stuff. 👍🏽

**Review by jeysal — APPROVED** [review]

**Pull request merged**